### ISSUE TITLE
Consider remove some patches

### ISSUE THREAD

**Issue by uazo** [bromite/bromite#2468]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.
- [X] This is a feature request for the Bromite browser; not the website nor F-Droid nor anything else.

### Is your feature request related to privacy?

No

### Is there a patch available for this feature somewhere?

N/A

### Describe the solution you would like

I think that

- [Remove offline measurement background task](https://github.com/bromite/bromite/blob/master/build/patches/Remove-offline-measurement-background-task.patch)
- [Unexpire tab groups flags](https://github.com/bromite/bromite/blob/master/build/patches/Unexpire-tab-groups-flags.patch)

can be dropped

### Describe alternatives you have considered

N/A

**Comment by csagan5** [comment]
> can be dropped

Why?

**Comment by uazo** [comment]
Of course, sorry

> [Remove offline measurement background task](https://github.com/bromite/bromite/blob/master/build/patches/Remove-offline-measurement-background-task.patch)

the patch removes the call to `OfflineMeasurementsBackgroundTask.clearPersistedDataFromPrefs` that clears
```
     * Keys used to store data for the OfflineMeasurementsBackgroundTask. The background task has
     * been removed, and these keys are just used to clear any persisted data.
```
> [Unexpire tab groups flags](https://github.com/bromite/bromite/blob/master/build/patches/Unexpire-tab-groups-flags.patch)

deactivation of `TabGroupsAndroid` and `TabGroupsUiImprovementsAndroid` leads to nothing except having to reactivate them by hand in order to make better use of the tabs

**Comment by csagan5** [comment]
> > [Unexpire tab groups flags](https://github.com/bromite/bromite/blob/master/build/patches/Unexpire-tab-groups-flags.patch)
> 
> deactivation of `TabGroupsAndroid` and `TabGroupsUiImprovementsAndroid` leads to nothing except having to reactivate them by hand in order to make better use of the tabs

I don't understand, if it leads to nothing what is there to reactivate?

**Comment by uazo** [comment]
> I don't understand, if it leads to nothing what is there to reactivate?

I'm saying that it's a useless patch, and that deactivating those flags accomplishes nothing, in fact, that it's almost better to leave them on. try it yourself.

**Comment by csagan5** [comment]
When enabling `#enable-tab-groups` and restarting (twice), tab groups become available.
When disabling that flag and restarting (twice), tab groups disappear and no new tab groups can be created.

**Comment by csagan5** [comment]
@uazo I am waiting for a reply on this, I have removed only the first one because the second one is effective.

**Comment by uazo** [comment]
sorry for the wait, bereavement in the family.
I don't understand your qualms about removing the other one as well, it doesn't seem to me to be anything problematic, on the contrary, it forces users to enter the flags and modify it.

**Comment by csagan5** [comment]
> sorry for the wait, bereavement in the family.

It is a free time project, no worries. Sorry to hear.

> I don't understand your qualms about removing the other one as well, it doesn't seem to me to be anything problematic, on the contrary, it forces users to enter the flags and modify it.

Groups are disabled by default and need to be enabled with this flag. It is working as intended. You wrote that the flags "lead to nothing", but they lead to something: when they are off, tab groups are disabled. When they are on, tab groups are enabled.

**Comment by csagan5** [comment]
Fixed in `108.0.5359.75`.
